cdhweb/people/models.py

```diff
@@ -129,7 +130,8 @@ class ProfileQuerySet(PublishedQuerySetMixin):
     student_titles = ['Graduate Fellow', 'Graduate Assistant',
                       'Undergraduate Assistant']
     #: memebership roles that indicate someone is an affiliate
-    project_roles = ['Project Director', 'Project Manager']
+    project_roles = ['Project Director',
+                     'Project Manager', 'Co-PI: Research Lead']
```

Comment:
Not sure that the list of director roles need to be defined on `Role` when we're defining other lists of roles here (esp. since that complicates the imports). Any reason not to just define it here? It doesn't seem to be used in the other file.

Reply:
sure, makes sense - that resolves the import issues too